ci/deploy.yaml

```diff
@@ -2,23 +2,23 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: theorf/google-cloud-sdk-helm
+    repository: ((image_registry))/eq-app-deploy-image
+    tag: ((deploy_image_version))
 params:
   SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
   PROJECT_ID:
   REGION: europe-west2
   DOCKER_REGISTRY:
   IMAGE_TAG: latest
   RUNNER_URL:
+
```

Comment:
Why the blank line? We don't use whitespace to separate other parts of the tasks.